pom.xml

```diff
@@ -415,7 +415,7 @@
 			<id>springNext</id>
 
 			<properties>
-				<spring.version>5.1.5.BUILD-SNAPSHOT</spring.version>
+				<spring.version>5.2.0.BUILD-SNAPSHOT</spring.version>
```

Comment:
Let's keep dependency changes out of this PR.

Reply:
I did it because of https://travis-ci.org/spring-projects/spring-vault/jobs/544499235.

This line is for the `springNext` profile only. Moreover, in line 29, we already have `<spring.version>5.2.0.M2</spring.version>`. I figured `springNext` should at least be a higher version, or the same one if there are no higher versions.

Happy to revert if that's what you want though.

Reply:
I see. I fixed that issue via 7460d5058b08baa2cf237706d2635f8a9fc27d8c.